### ISSUE TITLE
Rabbids Hollywood Arcade - Mouse stuck in top left corner

### ISSUE THREAD

**Issue by 1amgoingin** [argonlefou/DemulShooter#96]
When I try to setup mouse as P1 or P2 in demulshooter I can't move it in game, it gets recognised the crosshair is there and I can shoot but it is stuck in top left corner. This is also visible in demulshooter GUI. I will attach two images the one where I try to setup my joystick the crosshair is in the middle and when I move the stick it moves,  but when I switch to the mouse it just jumps to top left corner and does not move at all, I even tried to switch to other mouse i have, it's still the same. I am connecting mouse trough the usb port with a cable, so it's not Bluetooth. I thought it is because I have two monitors and maybe it is interfering with the software. I disconnected the other monitor and restarted the computer and it's still the same. 

![joystick](https://user-images.githubusercontent.com/22461272/207544891-96f6ba3b-6c86-4b43-9d21-ce1835a493ea.PNG)
![mouse](https://user-images.githubusercontent.com/22461272/207544898-190209c0-b08a-40d5-881f-814566c5ea3d.PNG)

**Comment by 1amgoingin** [comment]
I have now checked, I have the same problem with wild west shootout. mouse is stuck at top left corner, not quite top left, but close. and it seems like the crosshairs are not aligned with the joystick or the mouse. Especially at the top. So it has to do with y alignment I guess. On the bottom the crosshair is perfectly aligned but at the top it is quite out of sync on y axis. left and right is perfect. Maybe it's because of misalignment in resolution, have no clue.

**Comment by 1amgoingin** [comment]
Ok, I have managed to sync the crosshair by aligning my display resolution with the one in the registry keys. Now the mouse crosshair sits right up top left corner, hahaha. Does anyone have the same problem as I have?

**Comment by 1amgoingin** [comment]
I tried with DsDiag to find out what is going on with mouse, and it slightly changes the Rawinput data when I move the mouse, but the OnScreen data stays at 0x0 all the time, except when I yank the mouse hard then it kind of moves slightly to 1 or 2 depending on the direction I yank the mouse.

Here I managed to capture the screen when I yanked the mouse. there are also other information on the image like manufacutrer of the mouse but I don't think that's the problem really. 

By the way I am working on windows 10 pro version 10.0.19045 Build 19045

![DsDiag](https://user-images.githubusercontent.com/22461272/207561227-0d623fd8-466b-4fa3-ab0a-1cbff8efc9b0.PNG)
